### PR TITLE
Add TENANT_ID column to IDN_FED_AUTH_SESSION_MAPPING table

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
### Proposed changes in this pull request
* Fixes https://github.com/wso2/product-is/issues/13465
* Adding the TENANT_ID column to IDN_FED_AUTH_SESSION_MAPPING to store the tenant reference for the session id.

### CODE REVIEW

**Review by nilasini — APPROVED** [review]

**Comment by nilasini** [comment]
Did we create migration issue?

**Comment by ShanChathusanda93** [comment]
@nilasini I will create a migration issue and add it to here.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql [hunk 1]
```diff
@@ -1180,13 +1180,16 @@ CREATE TABLE IF NOT EXISTS IDN_OAUTH2_CIBA_REQUEST_SCOPES (
...
 CREATE TABLE IDN_FED_AUTH_SESSION_MAPPING (
+    ID SERIAL,
 	IDP_SESSION_ID VARCHAR(255) NOT NULL,
 	SESSION_ID VARCHAR(255) NOT NULL,
 	IDP_NAME VARCHAR(255) NOT NULL,
 	AUTHENTICATOR_ID VARCHAR(255),
 	PROTOCOL_TYPE VARCHAR(255),
 	TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-	PRIMARY KEY (IDP_SESSION_ID)
+	TENANT_ID INTEGER NOT NULL DEFAULT 0
```
Comment (gdiamond) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql:1190]: There is a missing comma here.  The line should end with a comma to become
```TENANT_ID INTEGER NOT NULL DEFAULT 0,```

Comment (gdiamond) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql:1190]: Never mind, It got fixed in #4105

**Pull request merged**